### ISSUE TITLE
KITTY object detection: performance measures plunge to almost zero at epoch #10

### ISSUE THREAD

**Issue by desertnaut** [NVIDIA/DIGITS#1315]
Hi all,

I am following the KITTY object detection demo, using the NVIDIA DIGITS 4 AMI in a g2.2xlarge instance. My settings are the ones suggested in the demo, with Batch size = 2 & Batch accumulation = 5 (in order to fit to my 4GB GPU memory), and Random seed = 1 (for reproducibility).

Thing is, at epoch 10, mAP falls abruptly from ~ 60 to almost zero. Here is the graph:

![kitty_1](https://cloud.githubusercontent.com/assets/7301435/20796566/56661bc6-b7df-11e6-87a9-cf45aac0c72c.PNG)

i.e. at the same time when learning rate falls from 1e-4 to 1e-5.

Is this expected (or at least not unexpected) behavior? And why exactly the learning rate falls - is it ADAM default behavior?

Any ideas?

I attach my `solver.prototext` and `caffe_output.log` files (zipped):

[KITTI.zip](https://github.com/NVIDIA/DIGITS/files/624785/KITTI.zip)

Many thanks in advance

**Comment by lukeyeager** [comment]
> Is this expected (or at least not unexpected) behavior?

I've seen something similar many times. I believe this is related to DetectNet rather than to Adam or Caffe. DetectNet is difficult to train.

> And why exactly the learning rate falls - is it ADAM default behavior?

You can change the learning rate policy in the form used to create a model. The default policy (which you appear to be using) drops by a factor of 0.1 three times during training.

**Comment by desertnaut** [comment]
Thanks. You are right about the learning rate policy.

I restarted training from epoch 9 with the same default policy, and the result is similar with what I have shown above. Still, it is a puzzle why - after all, a smaller learning rate is supposed to impose smaller changes to the net weights, not justifying the almost absolute drop in mAP...
